charts/public-service/README.md

```diff
@@ -122,6 +122,18 @@ $ helm upgrade --install \
     . 
 ```
 
+### Set up static IP address
```

Comment:
How is it related to the PR title?

Reply:
As I wrote in the comment, the change is actually only in that one commit.

Reply:
I would put it in a separate PR. It is a complete new paragraph for a topic not related to this PR.